src/tigris.ts

```diff
@@ -256,32 +258,84 @@ export class Tigris {
 	}
 
 	/**
-	 * Automatically provision Databases and Collections based on the directories
-	 * and {@link TigrisSchema} definitions in file system
+	 * Automatically create Project and create or update Collections.
+	 * Collection classes decorated with {@link TigrisCollection} decorator will be
+	 * created if not already existing. If Collection already exists, schema changes
+	 * will be applied, if any.
+	 */
+	public async syncCollections();
```

Comment:
SyncCollections could be confusing, as in is this something that is synchronizing the data of the collection. 

Why not registerSchemas or registerModels